recipes/Kili_fine_tuning_tutorial.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>it's a bit weird because in the first paragraph you already present the dataset we'll use, and then you say "we'll start by finding..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>The dataset is not mentioned earlier in the NB.</p>